.github/workflows/nightly.yaml

```diff
@@ -15,6 +15,12 @@ jobs:
         java: [ 11, 17 ]
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+      - name: Install github-cli
```

Comment:
I think github-cli, "gh" command, is already available in the runtime (macOS-12, ubuntu-22.04, windows-2022). Did you observe otherwise?

Reply:
You're correct. I've removed the setup steps now.